.github/workflows/review-bot.yml

```diff
@@ -18,9 +18,15 @@ jobs:
   review-approvals:
     runs-on: ubuntu-latest
     steps:
+      - name: Generate token
+        id: team_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: ${{ secrets.TEAM_APP_ID }}
+          private_key: ${{ secrets.TEAM_APP_KEY }}
       # !This must always point to main. 
       # Change it for the PRs but remember to change it back
       - uses: paritytech/review-bot@main
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          team-token: ${{ secrets.TEST_TEAM_TOKEN }}
+          team-token: ${{ steps.team_token.outputs.token }}
```

Comment:
Does it make sense to use its own (review bot) app? i am afraid how we gonna find out where the team app's id/key being used outside of [list-team-members](https://github.com/paritytech/list-team-members) ?

Reply:
It could make sense, but because it is being used for exactly the same requirements than list-team-members I thought that we could recycle that app.

For the new org we will need to create a new app anyways.

Reply:
then if this app dies or something else changes, would be useful to have the list of things app needs to access read/write. Just to recover quickly and for sure

Reply:
Yeah, I'll be adding it to the README eventually.

It just need `read: org` permissions. Nothing else.

You can find a comment telling that [here](https://github.com/paritytech/review-bot/blob/b1c2c41743590a996058d11b5b88621c6b3e95cd/src/github/teams.ts#L25)